libopendroneid/wifi.c

```diff
@@ -0,0 +1,340 @@
+/*
+Copyright (C) 2019 Intel Corporation
```

Comment:
I suggest you use another copyright and license header here.

Reply:
Sorry, maybe the license is okay but at least put your own name as the maintainer? (and change the company for the copyright).